src/System Application/App/Email/src/Email/Inbox/EmailInboxViewer.Page.al

```diff
@@ -0,0 +1,197 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Email;
+
+/// <summary>
+/// A page to view sent emails.
+/// </summary>
+page 81 "Email Inbox Viewer"
+{
+    PageType = Document;
+    SourceTable = "Email Inbox";
+    Caption = 'Email Inbox';
+    Permissions = tabledata "Email Inbox" = rd,
+                  tabledata "Email Message Attachment" = r;
+    DataCaptionExpression = '';
```

Comment:
Empty?

Reply:
Page is copied from Sent Emails Viewer. Probably will be deleted as well, per my other comment.